src/main/resources/avro/references.avdl

```diff
@@ -81,21 +81,12 @@ record ReferenceSet {
   string id;
 
   /**
-  The IDs of the `Reference` objects that are part of this set.
-
-  This field should be left null if the set contains a very large number of
-  `Reference`s. The `Reference`s in the set may also be obtained through a
```

Comment:
I'm happy to move this comment if it makes things clearer. Is it really necessary though? We don't explain that we use ``SearchVariants`` to get a ``Variant`` in the header comments for ``VariantSet`` or anywhere else in the object hierarchy.

 I think end-users are generally going to be lost if they are trying to use these API docs to use the API. End users should be using libraries (with their own documentation). This documentation is really just for client-side library developers, I think.